test/models/medium_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+class MediumTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+#validations
+  test 'A Medium Must Have a Title' do
+  	assert media(:flood).valid?
+  end
+
+# raise error if no title
+
+#   title must be a string
+#   maker name must be a string
+#   description must be a string
+
+# title cannot be just spaces
+# maker name cannot be just spaces
+# description cannot be just spaces
+
+
+
+
+  test 'A Medium Must Have a Maker' do
+  	assert_equal( media(:flood).maker, 'They Might Be Giants')
```

Comment:
This does not test that the medium must have a maker [to be valid], it only checks that the medium defined in the fixtures file has that particular maker.